invenio_rdm_records/records/mappings/v7/rdmrecords/records/record-v4.0.0.json

```diff
@@ -299,6 +299,42 @@
               }
             }
           },
+          "funding" : {
```

Comment:
A newer version (`v5.0.0`) should be created. It can be done after the feature is implemented.